packages/cli/lib/lib/webpack/sw_debug.js

```diff
@@ -0,0 +1,13 @@
+self.addEventListener('fetch', function (event) {
+    var isPostRequest = event.request.method === 'POST';
+    event.respondWith(fetch(event.request).catch(function (err) {
+        if (err instanceof TypeError) {
+            if (isPostRequest) {
+                console.log('PREACT-CLI-FRIENDLY-MESSAGE(IMPROVEMENT): We saw that your POST API call failed. This might fail for your users as well due to a network error, may be you should look into backgroundSync');
```

Comment:
Suggestion:

```js
console.log('⚛️Preact CLI development tip: A POST request just failed. This might fail for your users as well due to a network error. It may be worth exploring the backgroundSync API.');
```

```js
console.log('⚛️Preact CLI development tip: A GET request just failed. This might fail for your users as well due to a network error. It may be worth adding runtimeCaching to your Service Worker.');
```